documentation/getting_started/making_mods/unity_modding/index.md

```diff
@@ -0,0 +1,115 @@
+### Thunderkit
+To find the right version Find the Timberborn.exe and look at property's for it.   
+there Filevesion is show thats the version of Unity Thunderkit will need.   
+at writing 2022.1.6f1 is know to work
+
+### Mod unity version
+this will depend on what version timberAPI uses.  
+at writing 2021.3.1f1 is know to work
+
+
+## Instalation
+1. Download Zip [MinimalModSetup](https://github.com/KnatteAnka/MinimalModSetup)  
+1. Install [UnityHub](https://unity3d.com/get-unity/download)
+1. Open Unityhub/Installs and Click "Install Editor"
+1. Select Archive and click download archive and find the versions from above and install those
+
+## Create Project
+1. Unpack MinimalModSetup content and rename folder to your liking
+1. Open Unityhub/Projects and press Open and select folder from above
+
+## Thunderkit setup:
+
+### Steps to Import game Scripts
+1. Change version to use Thunderkit version. (Click on version in column and select the correct one)
+1. It will then Open the project, warning may show about version and just click continue
+
+1. Wait for Editor to open
+    * if question about Safe mode is shown select  
+    if it does press exit safe mode in the top left corner
+1. ThunderKit settings window should now show else its found:  
+    ![Thunderkit](/assets/images/assetripper/thunderkit.png)  
+    * When you have the thunderkit window open you can click on Show at startup to have it hidden as default.
+1. Select thunderkit settings and browse for the timberborn.exe and press import  
+![Thunderkit](/assets/images/assetripper/thunderkit_import.png)  
+    
+    
+
+## Modding Setup:   
+1. Close the project and change version to modding version as described above.
+1. It will then Open the project, warning may show about version and just click continue
+1. Wait for Editor to open, This may take some time
+    * If question about Safe mode is shown select  
+    if it does press exit safe mode in the top left corner 
+1. Try open Assets/Frog statue/FrogStatue.Folktails it should now show settings for the prefab example prefab name.
+    * if it does not redo the [Thunderkit](#steps-to-import-game-scripts)
+1. Up Top you have shortcuts to select mod and action do execute if non is shown:  
+"image"
+    1. Open Assets/Rebuild and Launch and uncheck and check Quick Access
+    1. do the same for any mods manifests
+1. you should now be able to select Rebuild and a mod in the list and press Execute to launch the game with that mod.
+    * even if it will crash because textures and meshes are missing.       
+
+## Update or add mod to Unity launch
+1. Download mod to update or add
+1. Open Packages\BepInExPack_Timberborn\BepInExPack_Timberborn\BepInEx\plugins
+1. Remove if mod folder if it already exists
+1. Copy downloaded mod to the folder
+1. next time you Open the project, it will import the mod, This may take some time
+
+## Export/Import package
+This can be used to transfer assets from Assetripped game to your mods project
```

Comment:
Maybe a chance to add a link to the "Exporting Game FIles" page here

Reply:
Good sugestion will do a test and we have to verify if it works or do a patch later to fix the link